kedro/framework/cli/starters.py

```diff
@@ -755,7 +755,7 @@ def _validate_config_file_inputs(config: dict[str, str]):
 
     input_tools = config.get("tools", "none")
     tools_validation_config = {
-        "regex_validator": r"^(all|none|(( )*\d*(,\d*)*(,( )*\d*)*( )*|( )*((\d+-\d+)|(\d+ - \d+))( )*))$",
+        "regex_validator": r"^(all|none|(( )*(\d*|\d( )?-( )?\d)(( )*,( )*(\d*|\d( )?-( )?\d))*( )*))$",
```

Comment:
You wrote a good explanation of the regex in the PR - can you actually include it here? Using `re.VERBOSE`:

https://docs.python.org/3/library/re.html#re.VERBOSE